### PR TITLE
Example: db reset

### PR DESCRIPTION
**Pull request by phh**
#2769

Example of `wp db reset` with the confirmation `--yes`

### CODE REVIEW

File: php/commands/db.php [hunk 1]
```diff
@@ -52,6 +52,10 @@ public function drop( $_, $assoc_args ) {
 	 *
 	 * [--yes]
 	 * : Answer yes to the confirmation message.
+	 * 
+	 * ## EXAMPLES
+	 *
+	 *     wp db reset --yes
```
Comment (danielbachhuber) [inline php/commands/db.php]: Can you make this a representational example?

Eg:

```
$ wp db reset --yes
Success: Database reset.
```

Comment (phh) [inline php/commands/db.php]: Updated (and added the same example for `wp db drop --yes`. Do you want me to keep the `$`? Seems like I couldn't find any examples containing it.

Comment (danielbachhuber) [inline php/commands/db.php]: > Do you want me to keep the `$`? Seems like I couldn't find any examples containing it.

Yes. The point of #2769 is to update the examples so they appear closer to how you'll actually see the commands run in the real world.

Comment (phh) [inline php/commands/db.php]: Aha - makes sense. Will keep it like that then :)

**Pull request merged**